requirements.txt

```diff
@@ -1,2 +1 @@
 ansible==1.6.6
```

Comment:
What happens if this is something like
``ansible>=1.9.4,<2.0`` instead of a fixed version?